collectors/0/goproc.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
```

Comment:
How come other collectors did not have this?

Reply:
Fixed